docs/demo/pythonSubmitDemo.py

```diff
@@ -77,24 +84,41 @@ class pythonProgramOptions:
 
     mode = 'BSE'                                                # evolving single stars (SSE) or binaries (BSE)?
 
-    grid_filename = 'Grid_demo.txt'
+    grid_filename = 'Grid_demo.txt'                                        # grid file name (e.g. 'mygrid.txt')
+
+    if grid_filename != None:
+        if compas_input_path_override == None:
+            grid_filename = os.getcwd() + '/' + grid_filename
+        else:
+            grid_filename = compas_input_path_override + '/' + grid_filename
```

Comment:
@themikelau In this section, it looks like you've added the possibility to use env var $COMPAS_INPUT_DIR_PATH to override the cwd path. This env var is new to me, is it something that's in use somewhere else? 

If not, the user already has to enter in the grid_filename, so I would think it would just be easier for the user to just enter in the path here, either as a relative path or a full path.

Reply:
Similar comment applies to logfile_defitions

Reply:
Hi Reinhold. Everything in pythonSubmitDemo.py was simply copied over from pythonSubmitDefault.py, and aren't changes I have personally made.

I am not familiar with `COMPAS_INPUT_DIR_PATH` either, but if it's not defined, then we will always have `        grid_filename = os.getcwd() + '/' + grid_filename`, which is fine, right? It doesn't seem like the `COMPAS_INPUT_DIR_PATH` functionality does anything bad.

Reply:
Oh sorry I have just missed Jeff's comment